app/src/androidTest/java/com/github/sdp_begreen/begreen/MainActivityTest.kt

```diff
@@ -50,4 +50,16 @@ class MainActivityTest {
         Intents.release()
     }
 
+    @Test
```

Comment:
This test seems to be good for me as it tests whether a button clicked in a Fragment leads to the correct Activity being started.